entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EMController.java

```diff
@@ -333,16 +331,16 @@ public EMController(EntityRecordService entityRecordService,
 	}
 
 			EntityRecord entityRecord = entityRecordService.retrieveEntityRecord(type, identifier,false);
-			return generateResponseEntity(profile, outFormat, contentType, entityRecord, HttpStatus.OK);
+			return generateResponseEntity(profile, outFormat, contentType, entityRecord, HttpStatus.OK, requestMethod);
 		}
 
 	private ResponseEntity<String> launchTaskAndRetrieveEntity(String type, String identifier,
-			EntityRecord entityRecord, String profile) throws Exception {
+			EntityRecord entityRecord, String profile, RequestMethod requestMethod, String contentType) throws Exception {
 		// launch synchronous update, then retrieve entity from DB afterwards
 		launchUpdateTask(Collections.singletonList(entityRecord.getEntityId()), false);
 		entityRecord = entityRecordService.retrieveEntityRecord(type, identifier, false);
 
-		return generateResponseEntity(profile, FormatTypes.jsonld, null, entityRecord, HttpStatus.ACCEPTED);
+		return generateResponseEntity(profile, FormatTypes.jsonld, contentType, entityRecord, HttpStatus.ACCEPTED, requestMethod);
```

Comment:
the FormatTypes is already a parameter for the generateResponse entity, and it defines the value for the contentType header, the parameter is redundant

Reply:
That makes sense. Improved.